internal/operators/ocs/validations.go

```diff
@@ -128,7 +103,18 @@ func (o *ocsValidator) ValidateRequirements(cluster *models.Cluster) (api.Valida
 	return api.Failure, status
 }
 
-func (o *ocsValidator) nodeResources(host *models.Host, cpuCount *int64, totalRAM *int64, diskCount *int64, hostsWithDisks *int64, insufficientHosts *[]string) (bool, error) {
+func checkInventoryAndNodeRequirements(o *operator, host *models.Host, cpuCount *int64, totalRAM *int64, diskCount *int64, hostsWithDisks *int64, insufficientHosts *[]string) (api.ValidationStatus, error) {
```

Comment:
Instead of passing `o *operator` , include that in function definition, so that you don't need to pass explicitly

Reply:
done